Framework/Geometry/src/Rendering/vtkGeometryCacheReader.cpp

```diff
@@ -7,16 +7,16 @@
 #include <Poco/Exception.h>
 #include <Poco/DOM/Element.h>
 
-using Poco::XML::DOMParser;
-using Poco::XML::Document;
-using Poco::XML::Element;
-
 #include "MantidKernel/Exception.h"
 #include "MantidKernel/Logger.h"
-#include "MantidGeometry/Objects/Object.h"
+#include "MantidGeometry/Objects/CSGObject.h"
```

Comment:
IObject.h

Reply:
Needed to prevent an incomplete type error.